File: Examples/VisualBasic/AsposePdfFacades/Forms/CopyInnerField.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf
Imports Aspose.Pdf.Facades

Namespace AsposePDFFacades.Forms
    Public Class CopyInnerField
        Public Shared Sub Run()
            Try
                ' ExStart:CopyInnerField
                ' The path to the documents directory.
                Dim dataDir As String = RunExamples.GetDataDir_AsposePdfFacades_Forms()
                ' Create FormEditor object
                Dim formEditor As New FormEditor()
                ' Open Document
                formEditor.BindPdf(dataDir & "CopyInnerField.pdf")
                ' Copy a field to another page
                formEditor.CopyInnerField("textfield", "newfieldname", 1)

                ' Close and save the output document
                formEditor.Save(dataDir & "CopyInnerField_out_.pdf")
                ' ExEnd:CopyInnerField
            Catch ex As Exception
                Console.WriteLine(ex.Message + vbLf & "This example will only work if you apply a valid Aspose License. You can purchase full license or get 30 day temporary license from http:// Www.aspose.com/purchase/default.aspx.")
            End Try

        End Sub
    End Class
End Namespace

File: Examples/CSharp/AsposePdfFacades/Forms/CopyInnerField.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2014 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;
using Aspose.Pdf.Facades;

namespace Aspose.Pdf.Examples.CSharp.AsposePDFFacades.Forms
{
    public class CopyInnerField
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Forms();
            //create FormEditor object
            FormEditor formEditor = new FormEditor();
            //Open Document
            formEditor.BindPdf(dataDir+ "CopyInnerField.pdf");
            //copy a field to another page
            formEditor.CopyInnerField("textfield", "newfieldname", 0);

            //close and save the output document
            formEditor.Save(dataDir+ "CopyInnerField_out.pdf");
            
            
        }
    }
}